### ISSUE TITLE
Codacy or Travis check for lint warnings in PRs?

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#1880]
We often realize belatedly that merged PRs contain lint warnings re: potential exceptions, increasing the risk for crashes in production versions after the PRs are merged in. It would be too big a burden to ask our reviewers to manually check the code they are merging for any lint warnings.

Anyone know if we can integrate this check into our Travis or Codacy CIs?

**Comment by chrisdebian** [comment]
I don't think it would be unreasonable to ask reviewers, but a good backstop would be to use code quality tools, as mentioned.

Chris
